cfg/phab.json

```diff
@@ -133,7 +133,7 @@
 
   // pre-subfilter triangulation options
   "triangulation": {
-    "method": 1,
+    "method": 5,
```

Comment:
Can we use more meaningful names, say, "L1", "L2", "Linf", etc. instead of numbers?

Reply:
The existing implementations of triangulation (using reprojection error) were just called "1" and "2". I think @agrawalparth10 decided to just follow that convention.

That said, now that there are five total methods implemented, making the method names and functions (currently named `Triangulate1` through `Triangulate5`) more descriptive is a good idea so that the code is self-documenting. How about something like `L1Angular`, `LinfAngular`, and `L2Reprojection`?

Reply:
yeah, I can change it!

Reply:
Please rename `Triangulate1` to `DirectLinearTransform`.